snoop/models.py

```diff
@@ -69,6 +69,9 @@ class Meta:
         # TODO: constraint does not apply to container=None rows
         unique_together = ('container', 'path')
 
+    def __str__(self):
+        return str(self.path)
```

Comment:
These will have collisions. You can have a `x.zip//one.txt` or `email.eml//one.txt` or just a disk file named `one.txt`. But all of them will show up as `one.txt`.

I think we should move the code that calculates the full path from [here](https://github.com/hoover/snoop/blob/master/snoop/digest.py#L56) into document (like we have done with `Document.open()`.